src/connect.js

```diff
@@ -145,9 +145,9 @@ function connect(mapStoreStateToProps, mapDispatchToProps, mergeProps, options =
 			}
 
 			/**
-			 * Returns the props data built from the store state, that should be
+			 * Returns the props data built from the store state that should be
 			 * passed to the wrapped component.
-			 * @param {bool} storeState
+			 * @param {!object} storeState
```

Comment:
Thanks a lot for fixing all those problems with the docs in this file :)

We're using [google's standard](https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler) for our api docs, and they use uppercase first letter for non primitive types, so in this case it should be `!Object` instead of `!object`. Can you fix this?